### PR TITLE
Validation Rules cleanup

### PR DESCRIPTION
**Pull request by suitedjimster**
Changes:
- Add validation names
- Fix isIndividual and isCompany for some vaiidation rules
- Update validatePerson and validateEntity functions
- Fix Chile (CL): Add labels for RUN, but shares the same validation as the RUT. RUN is for ind, RUT is for entity

### CODE REVIEW

File: src/ee/ik.ts [hunk 1]
```diff
@@ -65,6 +65,9 @@ export function ikCheckDate(value: string): boolean {
 }
 
 const impl: Validator = {
+  name: 'Estonian Personcal ID Number',
```
Comment (fronugget) [inline src/ee/ik.ts:68]: Personal

File: src/gt/cui.ts [hunk 2]
```diff
@@ -45,6 +45,9 @@ const citiesPerRegion: Record<string, number> = {
 };
 
 const impl: Validator = {
+  name: 'Guatemala identity Number',
```
Comment (fronugget) [inline src/gt/cui.ts:48]: Capital "I" in Identity

File: src/types.ts [hunk 3]
```diff
@@ -44,6 +44,25 @@ export type ValidateReturn = { error?: ValidationError } & (
...
+
+  /**
+   * The validator name, or ID's expansion name its local name
+   */
+  localName: string;
+
+  /**
+   * The short/acronym or abbreviation of the validator
+   */
+  abbreviation?: string;
```
Comment (fronugget) [inline src/types.ts:64]: What do we use to identify the type of ID when some don't have abbreviations? might make for an annoying impl. if we don't have something consistently to use reference the type, but I also see why inventing abbreviations might not be amazing given that this is a public package

Comment (suitedjimster) [reply src/types.ts:64]: fallback to localName.

**Comment by smaurydev** [comment]
> remove morocco ICE9; ICE is a 15 digit number.
Why ?

Like SIREN for SIRET in France, ICE9 allows to validate company numbers (first 9 digits of an ICE number) and not only establishments.

**Comment by suitedjimster** [comment]
@smaurydev 
If ICE9 is just a partial validator of ICE, then it really should exist outside the ICE validation. 
Also, the name ICE9 does not seem to be accurate since ICE is really a 15 digit ID.
I can't seem to find any documentation that uses the 9 digit company ID as a goverment ID.
Futhermore, I can't seem to find any strong validation rules for this company ID that would make it worthwhile to define a separate validator for it.

**Comment by suitedjimster** [comment]
@smaurydev I left the ICE9 in the rules.

**Review by koblas — APPROVED** [review]

**Pull request merged**